client/state/sites/domains/test/fixture.js

```diff
@@ -34,13 +36,15 @@ export const DOMAIN_PRIMARY = {
 	isPendingIcannVerification: false,
 	manualTransferRequired: false,
 	newRegistration: false,
+	name: 'retronevergiveup.me',
+	owner: 'John Doe',
 	partnerDomain: false,
 	pendingRegistration: false,
 	pendingRegistrationTime: '',
 	isPrimary: true,
 	isPrivate: false,
 	registrationDate: 'March 9, 2016',
-	type: 'mapping',
+	type: 'MAPPED',
```

Comment:
Umm, nope? The `getDomainType` fn returns these but I wanted to explicitly specify what it should contain afterwards.

Is there a better way? :)